### PR TITLE
Add travis

### PR DESCRIPTION
**Pull request by rcthomas**
I never get rebase right...

### CODE REVIEW

**Comment by bcfriesen** [comment]
This all looks good to me.

One nit-pick is that I have a personal distaste for merge commits like b278dab which merge the master branch into the feature branch as a synchronization mechanism. Those commits contain no information and clutter the commit history. The only merge commits in the history should be ones incoming to the master branch. To avoid merge commits, I prefer rebasing feature branches onto the master branch at whatever point in its history they would like to synchronize.

In short, I think this PR is good to merge, but will add something to the dev guide about merge commits.

**Comment by rcthomas** [comment]
That merge commit happened because I messed up the rebasing somehow.  I will merge and I will try to get it right next time... again.

**Comment by rcthomas** [comment]
Closes #4.
